### DISCUSSION TITLE
Upcoming Reader View Update

### DISCUSSION THREAD

**Discussion by ryspace** [tridactyl/tridactyl#4674] [Q&A] [answered]
I see that #102 has been closed. This is **AWESOME** news, and I cannot wait to be able to scroll with my home keys in reader view!

I see that you said it should be available "whenever the next beta is released" - is there any guesstimate as to when that might be? I noticed that the previous 4 tagged releases were (probably coincidentally) on a March/November/March/November "schedule"

I hope this isn't coming off as me being pushy. I'm just super excited about this and was curious, so I figured there wouldn't be any harm in asking.

Thanks for all the work you do on this project. I had to install Brave for something and had to use Vimium for it....it's okay I guess, but it doesn't even sniff Tridactyl. Even for the rare occasions where I'm using Brave for the one use case I have it for and I accidentally get sidetracked and do actual browsing with it, without fail, I end up getting frustrated within 1-2 minutes at missing features, aliases, commands, etc I've grown accustomed to using in Tridactyl.

~~When I'm no longer unemployed and struggling, your Patreon will be the first I subscribe to.~~ Scratch that - Just found out how sponsoring works, and I can figure out a way to spare a buck a month. Thanks again, and keep up the great work.

**Comment by bovine3dom** [comment, marked as answer]
Hi!

Thanks so much for your kind words and your sponsorship :) Best of luck in your job search.

I have some good news for you - the new reader mode is already out. If you're using the latest betas you just need to bind it with `:bind gr reader`

If you're not on the beta - it will tell you on the new tab page by saying Beta prominently - you can install it from https://tridactyl.cmcaine.co.uk/betas but make sure you disable the stable version on about:addons because the two versions will not coexist peacefully. If you have a complicated config there are instructions for migration on the wiki.

**Comment by ryspace** [comment]
Just installed it....awesome. Just gonna have to do some digging on how to tweak the style of the new reader page to my liking, but otherwise, it's perfect.

Thanks again

**Reply by bovine3dom** [reply]
There's currently no way within Tridactyl to do that - see #4657 - but you could patch it with userContent.css if you're feeling adventurous, see http://kb.mozillazine.org/UserContent.css

**Reply by ryspace** [reply]
One quick question @bovine3dom - Would it be possible to include the reading time estimate that's in the default Firefox reader mode?

**Reply by bovine3dom** [reply]
We would have to make one ourselves, it isn't included in the library. But it would be quite simple to do - I made an issue #4681 to track it
